### PR TITLE
Spelling issues

### LINKED ISSUE #908: Fix spelling issues in v4 docs
**Describe the bug**
V4 preview here: https://deploy-preview-819--docs-oceanprotocol.netlify.app/ har spelling issues. Thanks to @AnaLoznianu for pointing this out.

Pages to fix:

- [ ] https://deploy-preview-819--docs-oceanprotocol.netlify.app/concepts/architecture/

**To Reproduce**
NA

**Expected behavior**
All spelling, grammatical errors are fixed.

**Screenshots**
NA

**Desktop (please complete the following information):**
NA

**Smartphone (please complete the following information):**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by akshay-ap**
Fixes #908

Pages having spelling/grammatical issues:

- [x] https://deploy-preview-819--docs-oceanprotocol.netlify.app/concepts/architecture/
- [x] https://deploy-preview-819--docs-oceanprotocol.netlify.app/concepts/datanft-and-datatoken/
- [x] https://deploy-preview-819--docs-oceanprotocol.netlify.app/concepts/did-ddo/

Updated content:

- [x] https://deploy-preview-911--docs-oceanprotocol.netlify.app/concepts/architecture/
- [x] https://deploy-preview-911--docs-oceanprotocol.netlify.app/concepts/datanft-and-datatoken/
- [x] https://deploy-preview-911--docs-oceanprotocol.netlify.app/concepts/did-ddo/

### CODE REVIEW

**Comment by AnaLoznianu** [comment]
On the Architecture page, there is an image that has some mistakes.

File: content/concepts/architecture.md [hunk 1]
```diff
@@ -63,14 +60,14 @@ Complementary to Ocean Market, Ocean has reference code to ease building **third
...
 
-Ocean uses the Ethereum mainnet and other compatible networks as an **on-chain metadata store**, i.e. to store both DID and DDO. This means that once the write fee is paid, there are no further expenses or dev-ops work needed to ensure metadata availability into the future, aiding in the discoverability of data assets. It also simplifies integration with the rest of the Ocean system, which is Ethereum-based. Storage cost on Ethereum mainnet is not negligible, but not prohibitive and the other benefits are currently worth the tradeoff compared to alternatives.
+Ocean uses the Ethereum mainnet and other compatible networks as an **on-chain metadata store**, i.e. to store both DID and DDO. This means that once the transaction fee is paid, there are no further expenses or dev-ops work needed to ensure metadata availability into the future, aiding in the discoverability of data assets. It also simplifies integration with the rest of the Ocean system, which is Ethereum-based. Storage cost on Ethereum mainnet is not negligible, but not prohibitive and the other benefits are currently worth the tradeoff compared to alternatives.
```
Comment (AnaLoznianu) [inline content/concepts/architecture.md:68]: I think is devops, without a hyphen.
Also on the tradeoff, we need one...it is tradeoff

File: content/concepts/architecture.md [hunk 2]
```diff
@@ -63,14 +60,14 @@ Complementary to Ocean Market, Ocean has reference code to ease building **third
...
 
-Due to the permissionless, decentralized nature of data on Ethereum mainnet, any last-mile tool can access metadata. **Ocean Aquarius** supports different metadata fields for each different Ocean-based marketplace. Developers could also use [TheGraph](https://www.thegraph.com) to see metadata fields that are common across all marketplaces.
+Due to the permissionless, decentralized nature of data on the Ethereum mainnet, any last-mile tool can access metadata. **Ocean Aquarius** supports different metadata fields for each different Ocean-based marketplace. Developers could also use [The Graph](https://www.thegraph.com) to see metadata fields that are common across all marketplaces.
```
Comment (AnaLoznianu) [inline content/concepts/architecture.md:70]: last-mile without a hyphen please

**Comment by AnaLoznianu** [comment]
On the DataNFTs and Datatokens page the same, we need to update the image

**Comment by AnaLoznianu** [comment]
I noticed that, in some places we have MetaData, in others, we have Metadata. Let's use all over the places the same, please. Thank you!

**Review by AnaLoznianu — CHANGES_REQUESTED** [review]

**Comment by akshay-ap** [comment]
> On the Architecture page, there is an image that has some mistakes. Screenshot 2022-03-08 at 19 26 56

Pushed commit with updated image https://github.com/oceanprotocol/docs/pull/911/commits/54ec6159372ebd1fff95ca5c6a85a3b2cd34748e

**Comment by akshay-ap** [comment]
> On the DataNFTs and Datatokens page the same, we need to update the image Screenshot 2022-03-08 at 19 37 55
> 
> On the High-Level Behavior section, I'm not sure if what I've underlined here is an issue or not high-level

`datanft-and-datatoken.png` image fixed here: https://github.com/oceanprotocol/docs/pull/911/commits/a501c142dac2be0cd69253c08c561f4b628c36ac

**Pull request merged**